docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -99,7 +106,7 @@ Click the three dots next to `SSH configuration` as in the picture above.
 Create a new configuration by clicking the plus sign on the top left and providing:
 
 - `Host` - name of the Teleport Node.
-- `Port` - the Teleport Node's SSH port (3022 by default).
+- `Port` - the Teleport Node's SSH port (`3022` by default or `22` for OpenSSH servers).
```

Comment:
While we're here, should we update this to assume a reverse-tunnel connection through the Proxy, as is becoming our standard setup for documentation?

Reply:
3022 is still  used even with reverse tunnel.  Will leave as is.